vscode-trace-extension/src/utils/trace-server-status.ts

```diff
@@ -0,0 +1,24 @@
+import { StatusBarItem, ThemeColor } from 'vscode';
+
+export class TraceServerConnectionStatusService {
+
+    private statusBarItem: StatusBarItem;
+
+    public constructor(statusBarItem: StatusBarItem) {
+        this.statusBarItem = statusBarItem;
+        this.statusBarItem.hide();
+    }
+
+    public async render(status: boolean): Promise<void> {
+        if (status) {
+            this.statusBarItem.backgroundColor = new ThemeColor('statusBarItem.warningBackground');
+            this.statusBarItem.text = '$(check) Trace Server';
+            this.statusBarItem.tooltip = 'Trace Viewer: server found';
```

Comment:
Should `server` be `Server`?

Reply:
I'd prefer to keep `server`, here in this specific case given the colon use.